ui.apps/src/main/content/jcr_root/apps/etoolbox-backpack/assets/clientlibs/backpack/js/switchershowhide.js

```diff
@@ -0,0 +1,78 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+(function(document, $) {
+    "use strict";
+
+    /**
+     * Component initialization
+     */
+    $(document).on("foundation-contentloaded", function(e) {
+        $(".cq-dialog-checkbox-showhide").each( function() {
+            showHide($(this));
+        });
+    });
+
+    /**
+     * Listen to switcher changes
+     */
+    $(document).on("change", ".cq-dialog-checkbox-showhide", function(e) {
+        showHide($(this));
+    });
+
+    /**
+     * Show/Hide XPath or SQL2 elements, handle validation logic
+     * @param el represents switch element
+     */
+    function showHide(el){
```

Comment:
"Show or hide" something == "Toggle" in English. Consider naming this "toggleFilterElements" or sorts

Reply:
Function showHide was renamed to toggleFilterElements